fleet-local-community-playbook.yml

```diff
@@ -0,0 +1,32 @@
+site:
+  title: Fleet
+  url: /
+  start_page: v0.13@fleet:en:index.adoc
+
+content:
+  sources:
+    - url: ./
+      start_paths: [versions/v0.13]
+      branches: HEAD
+
+# Description: SUSE UI bundle
+ui:
+  bundle:
+    url: https://github.com/rancher/product-docs-ui/blob/main/build/ui-bundle.zip?raw=true
+    snapshot: true
+  supplemental_files: ./community-supp-files
```

Comment:
This is the correct expected path/value, but in it's current state you can't build because it's being [gitignored](https://github.com/rancher/fleet-product-docs/pull/96/files#diff-bc37d034bad564583790a46f19d807abfe519c5671395fd494d8cce506c42947R7) so this folder doesn't exist.